data/avgi/avgi 1 first contact.txt

```diff
@@ -0,0 +1,834 @@
+
+
+mission "Avgi: Scout Rescue"
+	name "Search and Rescue"
+	description "Find and evacuate the crew of the <npc>, which was last seen in the Twirl system."
+	source "Weledos"
+	passengers 12
+	destination "Peripheria"
+	waypoint "Twirl"
+	to offer
+		has "Avgi: Defend Ensemble 2: done"
+	to fail
+		has "avgi: warship intro"
+	on offer
+		conversation
+			`	You follow Kokkino's instructions to a prefab building in the spaceport, which looks to be some kind of command center. Kokkino is waiting for you by a table in the middle, which lights up, projecting a map of Avgi space in the air with some colored markings you can't read.`
+			`	"Explanation. We send scouting teams and patrols into the space controlled by the Aberrant regularly. Every expedition beyond Evanescence is essentially behind enemy lines, cut off from resupply and in many cases communication. As you might imagine, such adventures are fraught with danger, and it is unfortunately not unheard of for a scout ship to be lost."`
+			choice
+				`	"Yes, I've found that exploration is often dangerous."`
+					goto exploration
+				`	"Sounds like you should build stronger scout ships."`
+			`	"Agreement. The Polari are a fine line of vessels, but not one made for the environment of war. We have experimented with using the Prismaios as an intelligence platform in more contested environments, but it lacks the versatility and drone bays that has made the Polari so venerable in the first place."`
+				goto point
+			label exploration
+			`	He flicks his wings. "Regret. It is a sad truth, but those on the forefront of expanding our knowledge are often those who face the most danger from the unknown."`
+			label point
+			`	"But, to the point. A scout team, aboard the Polari-class scout 'Silver Albedo,' was sent to survey the Twirl system two weeks ago, but has missed its check-in by multiple days. To reach that star cluster and return would normally require a trip of weeks, all through perilous territory, and I am ashamed to say that we could be forced to consider writing off the scouts as lost. But I have heard that the visiting human, Captain <last>, is in possession of a new kind of faster-than-light drive, one that can travel without the need for starways. Tell me, is this true, question?"`
+			`	You nod.`
+			`	"Excitement. Then, the trip could be shortened to mere days, giving us the possibility of mounting a rescue. I will mark the location of the Twirl system on your map, though it is probable that the Silver Albedo is elsewhere in the star cluster. The highest priority is of course the safe recovery of its crew, though if you could recover any data or samples they have managed to gather, that would also be much appreciated."`
+				accept
+
+	npc assist
+		government "Avgi"
+		personality derelict mute pacifist target
+		system Whirlwind
+		ship "Polari (Derelict)" "Silver Albedo"
```

Comment:
As it stands, once you board the Silver Albedo it is repaired and can go wherever it wants. Perhaps make it self-destruct...?